int32 serialID

int32 driveDirection

int32 stop

int32 packIn
int32 packOut

int32 raiseBucketChain
int32 lowerBucketChain
int32 speedBucketChain

int32 raiseConveyorBelt
int32 lowerConveyorBelt
int32 speedConveyorBelt

int32 misc1
int32 misc2